modules/kwin.nix

```diff
@@ -633,7 +809,9 @@ in
 
       home.packages =
         with pkgs;
-        [ ] ++ lib.optionals (cfg.kwin.scripts.polonium.enable == true) [ polonium ];
+        [ ]
+        ++ lib.optionals (cfg.kwin.scripts.polonium.enable == true) [ polonium ]
+        ++ lib.optionals (cfg.kwin.scripts.krohnkite.enable == true) [ kdePackages.krohnkite ];
```

Comment:
```suggestion
        home.packages = lib.optionals cfg.kwin.scripts.polonium.enable [ pkgs.polonium ] ++ lib.optionals cfg.kwin.scripts.krohnkite.enable [ pkgs.kdePackages.krohnkite ];
```

Reply:
This wouldn't work, because the default values are `null`, and not a boolean

Reply:
Right! Sorry about that, I'm used to not having `null` in `enable` options.